scripts/insights-core-selinux.spec

```diff
@@ -0,0 +1,56 @@
+%define distro redhat
+%global debug_package %{nil}
+%global modulename insights_core
+%global selinuxtype targeted  #FIXME: which type? mls / targeted
+
+
+Name:           insights-core-selinux
+Version:        3.5.14
+Release:        1%{?dist}
+Summary:        Insights Core SELinux policy
+
+License:        Apache-2.0
+URL:            https://github.com/RedHatInsights/insights-core-selinux
+Source0:        %{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+Requires:       selinux-policy
+
+%description
+Insights Core SELinux policy module
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%pre
+%selinux_relabel_pre -s %{selinuxtype}
+
+%post
+%selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/%{selinuxtype}/%{modulename}.pp.bz2
+%selinux_relabel_post -s %{selinuxtype}
+
+%postun
+if [ $1 -eq 0 ]; then
+    %selinux_modules_uninstall -s %{selinuxtype} %{modulename}
+    %selinux_relabel_post -s %{selinuxtype}
+fi
+
+%build
+make
+
+
+%install
+install -D -pm 0644 %{modulename}.pp.bz2 %{buildroot}%{_datadir}/selinux/packages/%{selinuxtype}/%{modulename}.pp.bz2
+install -D -pm 0644 %{modulename}.if %{buildroot}%{_datadir}/selinux/devel/include/contrib/%{modulename}.if
+
+
+%files
+%{_datadir}/selinux/packages/%{selinuxtype}/%{modulename}.pp.*
+%{_datadir}/selinux/devel/include/contrib/%{modulename}.if
+%license LICENSE
+
```

Comment:
The following line is missing:
`%ghost %verify(not md5 size mode mtime) %{_sharedstatedir}/selinux/%{selinuxtype}/active/modules/200/%{modulename}`